bofire/data_models/features/molecular.py

```diff
@@ -1,75 +1,82 @@
 from typing import ClassVar, List, Literal, Optional, Tuple
 
 import pandas as pd
+from pydantic import Field
 
-from bofire.data_models.features.categorical import _CAT_SEP, TTransform
-from bofire.data_models.features.feature import Input
-from bofire.utils.cheminformatics import (
-    smiles2bag_of_characters,
-    smiles2fingerprints,
-    smiles2fragments,
-    smiles2mol,
-)
+from bofire.data_models.features.categorical import _CAT_SEP
+from bofire.data_models.features.feature import Input, TMolecularVals
+from bofire.data_models.molfeatures.api import AnyMolFeatures
+from bofire.utils.cheminformatics import smiles2mol
 
 
 class MolecularInput(Input):
     type: Literal["MolecularInput"] = "MolecularInput"
+    descriptor_values: TMolecularVals = Field(default_factory=list)
```

Comment:
For what this is needed? I am not getting it. From my point of view it can be calculated on the fly or and is not needed, or?

Reply:
The original idea was to save on compute time since mordred descriptors take longer to calculate and descriptor generation would be called twice to fit the model (first time to get the bounds, and the second time to transform the smiles strings)

But after reviewing again, my implementation wouldn't work for predictions for new molecules or different number of molecules. I've gotten rid of it! Good catch!

Reply:
Thx, I will do a proper review tmr.